includes/class-suggestion-engine.php

```
@@ -0,0 +1,352 @@
+		 * @return array Suggestion object.
+		 */
+		private static function make_boolean( string $metric, bool $passing, string $fix_action, string $description ): array {
+			return self::build(
+				$metric,
+				$passing ? 'pass' : 'fail',
+				'boolean',
+				$passing ? 'good' : 'poor',
+				$description,
+				$passing ? 'no_action_required' : $fix_action
+			);
+		}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'make_boolean'. (undefined)

(UnusedPrivateMethod)